### PR TITLE
Read replica content

### PR DESCRIPTION
**Pull request by danieltprice**
Previews:
- Read replicas (Introduction concept topic)
https://neon-next-git-dprice-read-only-replicas-neondatabase.vercel.app/docs/introduction/read-replicas
- Working with read replicas guide (screen caps to be added once available on prod)
https://neon-next-git-dprice-read-only-replicas-neondatabase.vercel.app/docs/guides/read-replica-guide

### CODE REVIEW

File: content/docs/introduction/read-replicas.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- **Throughput enhancement**: By distributing read requests among multiple read replicas, you can achieve higher throughput for both read-write and read-only workloads.
+- **Workload offloading**: Assign reporting or analytical workloads to a read replica to prevent any impact on the performance of read-write application workloads.
+- **Access control**: Provide read-only data access to certain users or applications that do not need write access.
+- **Resource customization**: Configure right-sized CPU and memory resources to cater to the specific needs of different users and applications.
+
+## Advantages
+
+Advantages of Neon's read replica feature include the following:
+
+1. **Efficient storage**: With read-only compute instances reading from the same source as your read-write computes, no additional storage is required to create a read replica. Data is neither duplicated nor replicated.
```
Comment (vadim2404) [inline content/docs/introduction/read-replicas.md:29]: it also worth mentioning the pricing. zero-overhead for storage price

File: content/docs/introduction/read-replicas.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+2. **Data consistency**: Read-write and read-only compute instances read data from a single source, ensuring the highest degree of data consistency.
```
Comment (vadim2404) [inline content/docs/introduction/read-replicas.md:30]: https://neondb.slack.com/archives/C04345PBJ49/p1685087949214039
look at these things too. Maybe it'll help to add more benefits

**Review by PlayLikeNeverB4 — APPROVED** [review]

File: content/docs/introduction/read-replicas.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/introduction/read-replicas.md:30]: Thank you. I will take a look to see what I can add.

Comment (danieltprice) [reply content/docs/introduction/read-replicas.md:30]: I think I have worked in these ideas now.

**Review by raoufchebri — APPROVED** [review]

**Pull request merged**